src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -73,6 +73,88 @@ def _core_registry_scenario(self, registry_name, resource_group, location,
         # test acr delete
         self.cmd('acr delete -n {} -g {}'.format(registry_name, resource_group))
 
+    @ResourceGroupPreparer
+    def test_acr_image_import(self, resource_group):
+        '''There are six test cases in the function, first of which should be executed in the 'ACR - CI' subscription and the rest are in 'ACR - Dev' subscription.
+        The test cases assume that there are source registry, source image and destination registry.'''
+
+        self.cmd("account set -s 'ACR - CI'")
```

Comment:
Remove hard coded subscription. As a general guideline, your test should be runnable by anyone.